lib/remote/apilistener.cpp

```diff
@@ -642,7 +642,9 @@ static const auto l_AppVersionInt (([]() -> unsigned long {
 })());
 
 static const auto l_MyCapabilities (
-	(uint_fast64_t)ApiCapabilities::ExecuteArbitraryCommand | (uint_fast64_t)ApiCapabilities::IfwApiCheckCommand
+	(uint_fast64_t)ApiCapabilities::ExecuteArbitraryCommand
+	| (uint_fast64_t)ApiCapabilities::IfwApiCheckCommand
+	| (uint_fast64_t)ApiCapabilities::HostChildrenInheritObjectAuthority
```

Comment:
Since enums are a constants anyway, why not make `MyCapabilities` a member of the enumeration class instead? That way you wouldn't need these kinda useless typecasting at all, and its a common way to define such enum types.
```cxx
...
MyCapabilities = ExecuteArbitraryCommand | IfwApiCheckCommand | HostChildrenInheritObjectAuthority
```

Reply:
> It was already there, I've just added another one.

I know it's already there! Just because something was already there doesn't mean you're never allowed to change it again. It get's with each new type added there even more ridiculous, hence my suggestion to change it, so that no one has to deal with it anymore.